proto/abci/types.proto

```diff
@@ -90,7 +90,7 @@ message RequestDeliverTx {
 }
 
 message RequestEndBlock {
-  int64 height = 1;
```

Comment:
This is a bit of an aside and perhaps not relevant to this PR, but don't you think that this file should have comments on what exactly the data structures and methods are for? Or is this covered elsewhere in the spec

Reply:
The displayed docs are located here: https://docs.tendermint.com/master/spec/p2p/messages/. We can add comments here, they will add value to us and tm-rs team